hazelcast/src/test/java/com/hazelcast/internal/partition/impl/MigrationCommitTest.java

```diff
@@ -296,9 +296,19 @@ public void shouldCommitMigrationWhenSourceFailsDuringCommit() {
 
         masterListener.other = hz2;
 
+        // hz1 and hz3 know the cluster members
         assertClusterSize(3, hz1, hz3);
+
+        // hz2 also  knows the cluster members
         assertClusterSizeEventually(3, hz2);
 
+        // In some rare occasions clusters member list is processed before the master information.
+        // Assert that hz2 and hz3 know the master
+        assertMasterAddressEventually(getAddress(hz1), hz2);
+        assertMasterAddressEventually(getAddress(hz1), hz3);
```

Comment:
Maybe we can use one statement
`assertMasterAddressEventually(getAddress(hz1), hz2, hz3);`

Reply:
I would like to see them separately in case it fails again.

Reply:
OK